test/lib/ansible_test/_internal/cli.py

```diff
@@ -309,6 +309,9 @@ def key_value_type(value):  # type: (str) -> t.Tuple[str, str]
     test.add_argument('--metadata',
                       help=argparse.SUPPRESS)
 
+    test.add_argument('--base-branch',
+                      help=argparse.SUPPRESS)
```

Comment:
Help text needed now that this is not an internal implementation detail:

```suggestion
                      help='base branch used for change detection')
```

Reply:
Definitely! I didn't add it yet since I wanted to wait to see whether you don't want `--base-branch` re-used for this.